src/main/scala/translating/PrettyPrinter.scala

```diff
@@ -288,6 +289,7 @@ class BasilIRPrettyPrinter(
   }
 
   override def vproc(p: Procedure): PPProg[Procedure] = {
+    p.sortBlocksRPO()
```

Comment:
I don't really like mutating the IR within the pretty-printer. Maybe this can be maintained as an invariant, but this could be difficult because lots of things can affect the RPO.